ansible_base/authentication/management/commands/authenticators.py

```diff
@@ -34,7 +35,7 @@ def handle(self, *args, **options):
                 try:
                     authenticator = Authenticator.objects.get(id=id)
                 except Authenticator.DoesNotExist:
-                    raise CommandError(f"Authenticator {id} does not exist")
+                    raise CommandError(_("Authenticator %(id) does not exist") % {"id": id})
```

Comment:
`(_("Authenticator %(id) does not exist") % {"id": id})` -> `(_("Authenticator %(id)s does not exist") % {"id": id})`